src/CollectSFDataGui/Server/Startup.cs

```diff
@@ -47,6 +53,12 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllersWithViews();
             services.AddRazorPages();
+            services.AddSingleton<IDataService, DataService>();
+
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "WebApplication1", Version = "v1" });
```

Comment:
can you give swagger endpoint a real name like CollecsfdataGuiEndpoint or something?

Reply:
suggestion is added in new commit